packages/bsky/src/services/moderation/index.ts

```diff
@@ -323,6 +323,17 @@ export class ModerationService {
     return subjectsDueForReversal
   }
 
+  async isSubjectSuspended(did: string): Promise<boolean> {
+    const res = await this.db.db
+      .selectFrom('moderation_subject_status')
+      .where('did', '=', did)
+      .where('suspendUntil', '>', new Date().toISOString())
```

Comment:
Could be nice to ensure we're talking about a suspension on an account takedown and not a record (despite that not being something that typically happens).  Would also allow you to drop the `limit(1)` since `(did, recordPath)` is unique.
```suggestion
      .where('did', '=', did)
      .where('recordPath', '=', '')
      .where('suspendUntil', '>', new Date().toISOString())
```

Reply:
ah cool good call - wasn't totally familiar with the new table schema 👌